### ISSUE TITLE
Documentation required to use jruby-base when compiling java extensions jruby-9.3.0.0+?

### ISSUE THREAD

**Issue by monkstone** [jruby/jruby#6856]
I picked up the availability of jruby-base in the 9.3.0.0 release notes, so I was keen to update my ruby-processing projects. So I tried pointing my polyglot build to jruby-base.pom but that did not work, whereas jruby-base.jar did. 

previously:-
```ruby
pom 'org.jruby:jruby:9.2.19.0'
```

now:-
```ruby
jar 'org.jruby:jruby-base:9.3.0.0'
```
In any event if we are now recommending use of jruby-base, then we should be changing documentation/[example](https://github.com/jruby/jruby-examples), I would happy to this if required?

**Comment by headius** [comment]
@monkstone Yeah we would like to start encouraging people to use jruby-base, since that is *just JRuby* and will not have odd dependency conflicts in a more heterogeneous environment. We got the artifact out there but forgot about doco. Anything you can contribute would be great!

**Comment by headius** [comment]
I've merged your PR. Not sure where else we need doco updates... perhaps in the wiki somewhere?
